src/app/components/patterns/DemoCard.tsx

```diff
@@ -127,13 +127,13 @@ export const DemoCard: React.FC<DemoCardProps> = (props): JSX.Element => {
                 {ionic && (
                     <DemoButton repository={repository} framework={'ionic'} url={ionic === true ? undefined : ionic} />
                 )}
-                {reactNative && (
+                {/*reactNative && (
```

Comment:
Only the react-native-design-patterns are not working. I'm not sure if we link out to anything else (component docs?) from a DemoCard, but you might want to double check that.

Reply:
I looked around and didn't see any react-native resources that have demos, this fix should be good to go.